README.md

```diff
@@ -1,2 +1,53 @@
-# Leaderboard
-This is a Repository where I will manage leaderboard having name and score of a person.
+# To-Do List
+
+>In this project I have setup an application that will consume API to create a game and save and fetch players's scores and built with Webpack. 
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- webpack
+- Linters
+- API Consumption
+
+## Getting Started
+To get a local copy up and running follow these simple example steps:
+ - Run "npm start" command at terminal to browse the page.
+
+### Prerequisites
+
+  - Web Browser to browse and following tools to debug and make changes (if any):
+      IDE eg (Atom/Visual studio code)
+      Set-up linter(HTML and CSS)
+      webpack
+
+### Setup
+  - Clone the github repository and set up linters and webpack
+
+
+## Author
+
+👤 **Muhammad Ashraf**
+
+- GitHub: [@for-ashraf](https://github.com/for-ashraf)
+- LinkedIn: [Muhammad Ashraf](https://www.linkedin.com/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
```

Comment:
- [ ] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```